gradebook/app/ui/pom.xml

```diff
@@ -79,6 +79,10 @@
       <groupId>org.sakaiproject.edu-services.gradebook</groupId>
       <artifactId>gradebook-service-hibernate</artifactId>
     </dependency>
+    <dependency>
+        <groupId>org.sakaiproject.assignment</groupId>
+        <artifactId>sakai-assignment-api</artifactId>
+    </dependency>
    <!-- <dependency>
```

Comment:
Do we really want this binding between the gradebook app and the assignment api?